packages/onboarding-ui/src/forms/OrganizationInfoForm/OrganizationInfoForm.tsx

```diff
@@ -28,9 +29,11 @@ type OrganizationInfoFormProps = {
   organizationIndustryOptions: SelectOptions;
   organizationSizeOptions: SelectOptions;
   countryOptions: SelectOptions;
+  onConfirmText?: string;
```

Comment:
For the sake of flexibility:

```suggestion
  onConfirmText?: ReactNode;
```

Reply:
I'll remember that!